docs/modules/ROOT/pages/how-tos/testing/integration/group-snapshots.adoc

```diff
@@ -0,0 +1,75 @@
+= Creating a group snapshot
+
+In {ProductName}, the integration service provides extra `group` snapshot to allow users to submit multiple related pull/merge requests under one application to their repos, have them tested together and have the CI results posted back to pull/merge requests, along with the component snapshot created for each component.
+
+== Procedures
+To have a `group` snapshot created for multiple pull/merge requests, complete the following steps:
+
+. Identify the application and components that are related and need to be updated.
+. Navigate to their git repos.
+. Create branches with the same branch name for all of these git repos.
+. Commit code changes to branches created above and create pull/merge requests.
+
+After every build pipelinerun succeeds and component snapshot is created, a group snapshot will be created containing the new built component container images and other components. The configured ItegrationTestScenario(s) will then be run with the component and group snapshot.
```

Comment:
The configured ITS(s) are those using the "group" context?

Reply:
`group` or `application` context ITS will all be run for group snapshot.

Reply:
Should we make that clearer?

Reply:
Updated. Thanks.